syntheseus/cli/search.py

```diff
@@ -67,8 +67,8 @@ class RetroStarConfig:
 class MCTSConfig:
     max_expansion_depth: int = 20
 
-    value_function_class: str = "ConstantNodeEvaluator"
-    value_function_kwargs: Dict[str, Any] = field(default_factory=lambda: {"constant": 0.5})
+    search_heuristic_class: str = "ConstantNodeEvaluator"
+    search_heuristic_kwargs: Dict[str, Any] = field(default_factory=lambda: {"constant": 0.5})
```

Comment:
In the context of MCTS, isn't "value function" a widely accepted term? Also, doesn't the use of this term here roughly match what it means in RL? I worry that switching from a usual term to a generic one may actually confuse someone who has used MCTS in the past.

Reply:
I agree. Let's leave "value function" as it is. I also don't think it is confusing.
We can add a comment or documentation to clarify that for some algorithms the value function is a (heuristic) negative cost function

Reply:
Yeah, I mean we could also rename "value function" to something else in Retro*-style algorithms, and keep it in MCTS-style algorithms, if that helps? I don't have a strong preference either way though.